readme.md

```diff
@@ -79,7 +79,7 @@ docker stop webgoat.net
 
 ### 2. Run locally using dotnet.exe (Kestrel)
 
-Requires .NET Core SDK 3.0 or later.
+Requires .NET 5 SDK.

```

Comment:
I am not sure if this is the right place for requirement notice. The same requirement has to be met while running on Docker or in any other way.